Fix #8 : Add story book

rray524 opened the pull request:
Fix #8 : Add story book

### Description

This pull request adds detailed instructions on how to run Storybook in a React project to the README.md file. This tool will help developers easily set up and use Storybook for developing and testing UI components in isolation. This pr is also created for adding npm package of storybook.

### Type of Change

* Add storybook npm package 
*  update readme about this storybook & how to run it

rt4914 commented on .gitignore:
```diff
@@ -24,3 +24,5 @@ dist-ssr
 
 .DS_Store
+
+*storybook.log
```
Add blank line here at the end

rray524 replied:
added.

rt4914 reviewed (changes requested):
For now this is fin.e But we should add an issue to mention steps on how to run storybook in Readme.md file

Once that issue is filed, mention it here so that I can approve this and we can merge.

rray524 commented:
> For now this is fin.e But we should add an issue to mention steps on how to run storybook in Readme.md file
> 
> Once that issue is filed, mention it here so that I can approve this and we can merge.

I updated read me with a proper running command, which can be copied from it and a small intro of storybook. Thanks @rt4914

rt4914 commented:
> > For now this is fin.e But we should add an issue to mention steps on how to run storybook in Readme.md file
> > Once that issue is filed, mention it here so that I can approve this and we can merge.
> 
> I updated read me with a proper running command, which can be copied from it and a small intro of storybook. Thanks @rt4914

@rray524 Where did you update the README ? I am unable to see that. 


Also README.md file is currently getting developed so that's why I have mentioned that you should create an issue once that README.md file gets created correctly, then we can mention the updated steps in that file by create a new PR.

Also, merge your branch with latest main to resolve merge conflicts.

rray524 commented:
> > > For now this is fin.e But we should add an issue to mention steps on how to run storybook in Readme.md file
> > > Once that issue is filed, mention it here so that I can approve this and we can merge.
> > 
> > 
> > I updated read me with a proper running command, which can be copied from it and a small intro of storybook. Thanks @rt4914
> 
> @rray524 Where did you update the README ? I am unable to see that.
> 
> Also README.md file is currently getting developed so that's why I have mentioned that you should create an issue once that README.md file gets created correctly, then we can mention the updated steps in that file by create a new PR.
> 
> Also, merge your branch with latest main to resolve merge conflicts.

@rt4914 here is the readme.md file where I added how to run storybook & intro of storybook:
https://github.com/rt4914/KlinicCon-Frontend/blob/8614947832d2b7aecb809fea72232487ff0b9765/README.md

rt4914 commented:
> > > > For now this is fin.e But we should add an issue to mention steps on how to run storybook in Readme.md file
> > > > Once that issue is filed, mention it here so that I can approve this and we can merge.
> > > 
> > > 
> > > I updated read me with a proper running command, which can be copied from it and a small intro of storybook. Thanks @rt4914
> > 
> > 
> > @rray524 Where did you update the README ? I am unable to see that.
> > Also README.md file is currently getting developed so that's why I have mentioned that you should create an issue once that README.md file gets created correctly, then we can mention the updated steps in that file by create a new PR.
> > Also, merge your branch with latest main to resolve merge conflicts.
> 
> @rt4914 here is the readme.md file where I added how to run storybook & intro of storybook: https://github.com/rt4914/KlinicCon-Frontend/blob/8614947832d2b7aecb809fea72232487ff0b9765/README.md so I am removing the steps now and will later, once readme.md full get resolved. **Removed merging conflict too.** Thanks.

@rray524 So if you look at that README.md file, its just a sample file and because of that I was suggesting you to create an issue. But now we have merged/updated the README.md file so you can add this correctly directly in README.md file
https://github.com/rt4914/KlinicCon-Frontend?tab=readme-ov-file#installation
Please add this correctly in installation-frontend section clearly mentioning that its a an optional step to run a storybook.

Also, I was unable to review it because you never assigned it to me. So please make sure you `Assignee` PRs to me.

rray524 commented:
> > > > > For now this is fin.e But we should add an issue to mention steps on how to run storybook in Readme.md file
> > > > > Once that issue is filed, mention it here so that I can approve this and we can merge.
> > > > 
> > > > 
> > > > I updated read me with a proper running command, which can be copied from it and a small intro of storybook. Thanks @rt4914
> > > 
> > > 
> > > @rray524 Where did you update the README ? I am unable to see that.
> > > Also README.md file is currently getting developed so that's why I have mentioned that you should create an issue once that README.md file gets created correctly, then we can mention the updated steps in that file by create a new PR.
> > > Also, merge your branch with latest main to resolve merge conflicts.
> > 
> > 
> > @rt4914 here is the readme.md file where I added how to run storybook & intro of storybook: https://github.com/rt4914/KlinicCon-Frontend/blob/8614947832d2b7aecb809fea72232487ff0b9765/README.md so I am removing the steps now and will later, once readme.md full get resolved. **Removed merging conflict too.** Thanks.
> 
> @rray524 So if you look at that README.md file, its just a sample file and because of that I was suggesting you to create an issue. But now we have merged/updated the README.md file so you can add this correctly directly in README.md file https://github.com/rt4914/KlinicCon-Frontend?tab=readme-ov-file#installation Please add this correctly in installation-frontend section clearly mentioning that its a an optional step to run a storybook.
> 
> Also, I was unable to review it because you never assigned it to me. So please make sure you `Assignee` PRs to me.

hi @rt4914 I have assigned you to review. I have made changes on readme.md . thanks

rt4914 reviewed (approved):
LGTM, Thanks.
